x-pack/plugins/transform/public/app/sections/transform_management/components/transform_list/columns.tsx

```diff
@@ -242,7 +224,7 @@ export const getColumns = (
     {
       name: i18n.translate('xpack.transform.tableActionLabel', { defaultMessage: 'Actions' }),
       actions,
-      width: '200px',
+      width: '100px',
```

Comment:
I think we can drop the separate 'Start' action and just have it available in the dropdown menu shown from the `...` button.